deployment/.env.template

```diff
@@ -15,17 +15,21 @@ PFS_HOST=0.0.0.0
 PFS_KEYSTORE_FILE=/keystore/<FILENAME>
 PFS_PASSWORD=<PASSWORD>
 PFS_LOG_LEVEL=DEBUG
+# PFS_ACCEPT_DISCLAIMER=TRUE
 
 MS_KEYSTORE_FILE=/keystore/<FILENAME>
 MS_PASSWORD=<PASSWORD>
 MS_LOG_LEVEL=DEBUG
+# MS_ACCEPT_DISCLAIMER=TRUE
 
 MSRC_KEYSTORE_FILE=/keystore/<FILENAME>
 MSRC_PASSWORD=<PASSWORD>
 MSRC_LOG_LEVEL=DEBUG
+# MSRC_ACCEPT_DISCLAIMER=TRUE
 
 MS_BACKUP_KEYSTORE_FILE=/keystore/<FILENAME_BACKUP>
 MS_BACKUP_PASSWORD=<PASSWORD_BACKUP>
+# MS_ACCEPT_DISCLAIMER=TRUE
```

Comment:
I don't think we need that one.

Reply:
ok removed. How does the backup MS start then?